include/common_structs.h

```diff
@@ -172,7 +171,7 @@ typedef struct {
     /* 0x0078 */ s16 unk_078;
     /* 0x007A */ s16 unk_07A;
     /* 0x007C */ s32 unk_07C;
-    /* 0x0080 */ f32 unk_080;
+    /* 0x0080 */ f32 boostVelocity;
```

Comment:
Played around with this and I think a better term would be `boostPower` or `boostStrength`. When the player gets a boost, this field ramps up to a value and stays there until `boostTimer` reaches zero, and then it ramps down to 1. Once it reaches one the boost ends. It doesn't appear to be a direct measure of velocity

Reply:
Ah, I remember exploring this one now. Yea, I'm fine with either boostPower or boostStrenth.